lib/menu.rb

```diff
@@ -0,0 +1,30 @@
+class Menu
+
+  attr_accessor :dishes
+
+  def initialize(dishes)
+
+    @dishes = dishes
+  end
+
+  def print
+    @dishes.map do |dish, price|
+      "#{dish}: £#{price}"
+
+    end.join(", ")
+    
+  end
+
+  def has_dish?(dish)
```

Comment:
Really good extracting this check to its own method and also in a Ruby conventional way.